config.md

```diff
@@ -0,0 +1,57 @@
+* mesos.hdfs.role - Used to determine the mesos role this framework will use.  The default is "*".
+* mesos.hdfs.mesosdns - true if mesos-dns is used.  The default is false.
+* mesos.hdfs.mesosdns.domain - When using mesos-dns, this value is the suffix used by mesos-dns.  The default is "mesos".
+* mesos.native.library - The location of libmesos library.  The default is "/usr/local/lib/libmesos.so"
+* mesos.hdfs.journalnode.count - The number of journal nodes the scheduler will maintain. The default is 3.
+* mesos.hdfs.data.dir -  The location to store data on the slaves.  The default is "/var/lib/hdfs/data".
+* mesos.hdfs.domain.socket.dir - The location used for a local socket used by the data nodes.  The default is "/var/run/hadoop-hdfs".
+* mesos.hdfs.backup.dir - The location to replicated data to as a backup.  The default is blank.
+* mesos.hdfs.native-hadoop-binaries -  This is true if hdfs is pre-installed on the slaves.  This will result in no distribution of binaries to the slaves.  It will also mean that no xml configure refresh will be provided to the slaves.  The default is false.
+* mesos.hdfs.framework.mnt.path - If native-hadoop-binaries == false, this is the location a symlink will be provided to execute hdfs commands on the slave.  The default is "/opt/mesosphere"
+* mesos.hdfs.state.zk - The zookeeper that the scheduler will use to store state.  The default is "localhost:2181"
+* mesos.master.uri - The zookeeper or mesos-master url that will be used to discover the mesos-master for scheduler registration. The default is "localhost:2181"
+* mesos.hdfs.zkfc.ha.zookeeper.quorum - The zookeeper that HDFS (not the framework) will use for HA mode.  The default is "localhost:2181"
+
+There are additional configurations for executor jvm and resource management of the nodes.
+
+## System Environment Variables
+
+All of configuration flags previous defined can be override with system environment variables.  The format to use to over a variable is to
+upper case the string and replace dots (".") with underscores ("_").  so to override the `mesos.hdfs.framework.name`, the value is `MESOS_HDFS_FRAMEWORK_NAME=unicorn".
+To use this value, export the value, then start the scheduler.  If a value is overridden by the system environment variable it will be propagated to
+the executors.
+
+## Custom Configurations
+
+### Mesos-DNS custom configuration
+You can see the example configuration in the `example-conf/dcos` directory. Since Mesos-DNS provides native bindings for master detection, we can simply use those names in our mesos and hdfs configurations. The example configuration assumes your Mesos masters and your zookeeper nodes are colocated. If they aren't you'll need to specify your zookeeper nodes separately. Also, note that you are using the example in `example-conf/dcos`, the `mesos.hdfs.native-hadoop-binaries` property needs to be set to `false` if your HDFS binaries are not predistributed.
```

Comment:
This is copy pasted from elsewhere.  There was a typo in the other version, so fix it here too.